server/app/services/question/PrimaryApplicantInfoTag.java

```diff
@@ -0,0 +1,105 @@
+package services.question;
+
+import com.google.common.collect.ImmutableList;
+import models.QuestionTag;
+import services.question.types.QuestionType;
+
+/**
+ * Defines tags that can be applied to a question to denote the answer is a primary piece of
+ * information about the applicant. These are 1:1 with particular QuestionTags.
+ */
+public enum PrimaryApplicantInfoTag {
+  APPLICANT_DOB(
+      QuestionTag.PRIMARY_APPLICANT_DOB,
+      QuestionType.DATE,
+      "primaryApplicantInfoDob",
+      "Applicant Date Of Birth",
+      "Setting this property will allow the answer to be pre-populated with the"
+          + " applicant's date of birth if a TI created this applicant."),
+  APPLICANT_EMAIL(
+      QuestionTag.PRIMARY_APPLICANT_EMAIL,
+      QuestionType.EMAIL,
+      "primaryApplicantInfoEmail",
+      "Applicant Email Address",
+      "Setting this property will allow the email address collected from this"
```

Comment:
question (blocking): do we want to include that email, name, and phone will also be pre-populated if the client is added by a TI or is that too much info?